health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserUpload.js

```
@@ -366,6 +366,57 @@ const UserUpload = React.memo(() => {
+            campaignId: id,
+            additionalDetails: {
+              source: "microplan"
+            }
+          }
+        }
+      }
+      try {
+        await axios.post(reqCriteria.url, reqCriteria.body);
+      } catch (error) {
+        var errorLabel;
+        if (error?.response && error?.response?.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)